packages/components/pie-chip/src/chip.scss

```diff
@@ -50,6 +17,9 @@
     // Pie Webc Icon var that is used to ensure the correctly sized icon passed in a slot
     --icon-display-override: block;
 
+    // this token is needed for interactive-states mixin
+    --chip-bg-transparent: transparent;
```

Comment:
I get why this is necessary but it would be easier if we had a design token for `transparent` 🤔

Reply:
Yes, and I think we used to have it but it wasn't used in code so we removed it... But probably worth adding it back even if it will stay as "transparent" for the time being. I'll write a follow up ticket to add the token and update the usage